_posts/en/newsletters/2021-12-22-newsletter.md

```diff
@@ -0,0 +1,665 @@
+  offered experimental support for wallets that natively use [output
+  script descriptors][topic descriptors].
+
+- [Rust Bitcoin 0.26.0][] included support for signet, version 2 address
+  announcement messages, and improvements to [PSBT][topic psbt]
+  handling.
+
+- [LND 0.12.0-beta][] included support for using [watchtowers][topic
+  watchtowers] with [anchor outputs][topic anchor outputs] and added a
+  new `psbt` wallet subcommand for working with [PSBTs][topic psbt].
+
+- [HWI 2.0.0][] contained support for multisig on the BitBox02, improved
+  documentation, and support for paying `OP_RETURN` outputs with a
+  Trezor.
+
+- [C-Lightning 0.10.0][] contained a number of enhancements to its API
+  and experimental support for [dual funding][topic dual funding].
+
+- [BTCPay Server 1.1.0][] included [Lightning Loop][news53 lightning
+  loop] support, added [WebAuthN/FIDO2][fido2 website] as a two-factor
+  authentication option, made various UI improvements, and marked a
+  switch to using [semantic versioning][semantic versioning website] for
+  version numbers moving forward.
+
+- [Eclair 0.6.0][] contained several improvements that enhanced user
+  security and privacy.  It also provided compatibility with future
+  software that may use [bech32m][topic bech32] addresses.
+
+- [LND 0.13.0-beta][] improved feerate management by making [anchor
+  outputs][topic anchor outputs] the default commitment transaction
+  format, added support for using a pruned Bitcoin full node, allowed
+  receiving and sending payments using Atomic MultiPath ([AMP][topic
+  multipath payments]), and increased LND's [PSBT][topic psbt]
+  capabilities
+
+- [Bitcoin Core 22.0][] included support for [I2P][topic anonymity
+  networks] connections, removed support for [version 2 Tor][topic
+  anonymity networks] connections, and enhanced support for [hardware
+  wallets][topic hwi].
+
+- [BDK 0.12.0][] added the ability to store data using Sqlite.
+
+- [LND 0.14.0][] included additional [eclipse attack][topic eclipse
+  attacks] protection (see [Newsletter #164][news164 ping]), remote
+  database support ([Newsletter #157][news157 db]), faster pathfinding
+  ([Newsletter #170][news170 path]), improvements for users of Lightning
+  Pool ([Newsletter #172][news172 pool]), and reusable [AMP][topic amp]
+  invoices ([Newsletter #173][news173 amp]).
+
+- [BDK 0.14.0][] simplified adding an `OP_RETURN` output to a
+  transaction and contained improvements for sending payments to
+  [bech32m][topic bech32] addresses for taproot.
+
+</div>
+
+## June
+
+A new [analysis][csb] discussed in June described an alternative way for
+miners to select which transactions they want to include in the blocks
+they create.  The new method is predicted to slightly increase miner
+revenue in the short term.  In the long-term, wallets aware of the new
+method will be able to collaborate when [CPFP fee bumping][topic cpfp]
```

Comment:
Candidate set-based blockbuilding would first need to be adopted by some miners, and that would likely only happen after it gets implemented in Bitcoin Core, so this feels a bit too optimistic.

```suggestion
method would be able to collaborate when [CPFP fee bumping][topic cpfp]
```